lib/ddtrace/monkey.rb

```diff
@@ -5,14 +5,21 @@
 # patching code, which is required on demand, when patching.
 require 'ddtrace/contrib/active_record/patcher'
 require 'ddtrace/contrib/elasticsearch/patcher'
-require 'ddtrace/contrib/http/patcher'
+require 'ddtrace/contrib/grape/patcher'
 require 'ddtrace/contrib/redis/patcher'
+require 'ddtrace/contrib/http/patcher'
```

Comment:
Was there an issue with the order? (just curious, I'm not requiring any change here)